packages/@contentful--app-scripts/lib/create-app-definition/create-app-definition.js

```diff
@@ -64,6 +65,11 @@ async function createAppDefinition(accessToken, appDefinitionSettings = { locati
   ]);
   const selectedOrg = organizations.find((org) => org.value === organizationId);
 
+  await cacheEnvVars({
+    'CONTENTFUL_ORG_ID': organizationId,
+    'CONTENTFUL_ACCESS_TOKEN': accessToken,
+  });
+
```

Comment:
Are we even reading the variables from the environment once we write them? 🤔 Or is it part of a followup?

Reply:
Lets do a follow up as discussed.